index.html

```diff
@@ -10,6 +10,7 @@
     <body>
         <p>Hello World!</p>
 
+        <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/system.js"></script>
```

Comment:
Script tag SystemJS, don't put it into your webpack bundle. This solves the issue.